sdk/servicebus/azure-servicebus/tests/mgmt_tests/async/test_mgmt_rules_async.py

```diff
@@ -27,9 +27,13 @@
 class TestServiceBusAdministrationClientRuleAsync(AzureMgmtRecordedTestCase):
     @ServiceBusPreparer()
     @recorded_by_proxy_async
-    async def test_async_mgmt_rule_create(self, servicebus_connection_str, **kwargs):
+    async def test_async_mgmt_rule_create(self, servicebus_fully_qualified_namespace, **kwargs):
```

Comment:
here we are passing through the fully_qualified_namespace -- can we do the same on the non-mgmt tests?

Reply:
will have to leave that for another PR. mgmt tests pull in their info from a preparer, while the non-mgmt get their info during run time. Would need some mods to that preparer.